packages/common/src/node-status-interface.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * A JSON object containing various diagnostic and introspection information from a running
+ * Ceramic node.
+ */
+export interface NodeStatusResponse {
+  /**
+   * A random UUID that is generated each time a node starts up.  Can be used to detect when
+   * a node restarts.
+   */
+  runId: string
+
+  /**
+   * The Ceramic network the node is connected to.
+   */
+  network: string
+
+  /**
+   * Anchoring-related information.
+   */
+  anchor: AnchorNodeStatus
+
+  /**
+   * Information about the connected IPFS node.
+   */
+  ipfs: IpfsNodeStatus
+}
+
+/**
+ * IPFS sub-object within NodeStatusResponse
+ */
+export interface IpfsNodeStatus {
+  /**
+   * PeerId of the connected ipfs node
+   */
+  peerId: string
+
+  /**
+   * IPFS Swarm multiaddrs of the connected ipfs node
+   */
+  addresses: Array<string>
+}
+
+export interface AnchorNodeStatus {
```

Comment:
```suggestion
export interface AnchorNodeStatus {
  /**
   * The ethereum chainId used for anchors.
   */
  chainId: string
```

What about adding this as well?